integration_test/gce/gce_testing.go

```diff
@@ -1273,6 +1273,16 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 		return nil, err
 	}
 
+	if isUbuntu(vm.Platform) {
+		// Taken from https://askubuntu.com/a/1039155 to fix b/310644585.
+		// This is unnecessary for newer versions of apt because apt was fixed
+		// (https://bugs.debian.org/754103) to wait for the lock instead of
+		// failing out immediately.
+		if _, err := RunRemotely(ctx, logger, vm, "", "sudo systemctl disable apt-daily.service && sudo systemctl disable apt-daily.timer && sudo systemctl disable apt-daily-upgrade.timer && sudo systemctl disable apt-daily-upgrade.service"); err != nil {
```

Comment:
This is still racy because the service might be running when you try to disable it